### ISSUE TITLE
Made with :heart:

### ISSUE THREAD

**Issue by tylerlaprade** [githubnext/monaspace#224]
<img width="407" alt="image" src="https://github.com/githubnext/monaspace/assets/5475199/14b3a2b7-ec10-4214-8505-78f0d3b8b2db">

The ❤️ emoji is not rendered on the website.

**Comment by kenmcd** [comment]
Has nothing to do with this font - there are no emojis in the font.
What you see for that emoji shortcode depends on the website, the browser, the operating system.

What I see on Win10 in Firefox is `/heavyblackheart` or `U+2764` in the Segoe UI Emoji font.
You may see something different.

If you want something specific, do not use a shortcode.
Specify the font and the codepoint.

**Comment by rileycran** [comment]
I believe this is purposeful as an inside copywriting joke by @idan  😉

**Comment by tylerlaprade** [comment]
@kenmcd, I am referring to Monaspace's website itself.

**Comment by kenmcd** [comment]
It appears `githubnext.com` does not support emoji shortcodes.

**Comment by idan** [comment]
@tylerlaprade @rileycran is correct, a couple of years ago at GitHub Universe, Kyle Daigle did a number of [GitHub Next demos at a session titled `Building with :heart:`](https://www.youtube.com/watch?v=4NuK415HYTI) so we riffed on that.

tl;dr it's a joke, no emojis are supposed to be rendered there
